docs/spec.md

```diff
@@ -0,0 +1,351 @@
+# Specification: Ballerina Redis Library
+
+_Authors_: @NipunaRanasinghe
+_Reviewers_: @AyeshLK @AzeemMuzammil
+_Created_: 2022/02/19  
+_Updated_: 2022/02/19  
+_Edition_: Swan Lake
+
+## Introduction
+
+This specification outlines the functionality provided by the Redis connector for
```

Comment:
Shall we adopt the following structure [1] for the introduction section as all the other hand-written connectors follow the same structure.

[1] - [https://github.com/ballerina-platform/module-ballerina-graphql/blob/master/docs/spec/spec.md#introduction](https://github.com/ballerina-platform/module-ballerina-graphql/blob/master/docs/spec/spec.md#introduction)

Reply:
+1 Updated with Changed with https://github.com/ballerina-platform/module-ballerinax-redis/pull/219/commits/5aec5d13ea49f791379343d8759d3c4819f987ad.